### PR TITLE
Feature/math test

### PR DESCRIPTION
**Pull request by hakuturu583**
## Types of PR

- [ ] New Features
- [x] Upgrade of existing features
- [x] Bugfix

## Description

- Add test cases in math test.
- Remove unused message types.
- Fix problems in calculating maximum curventure.

## How to review this PR.
please check passing all test cases.

### CODE REVIEW

File: simulation/traffic_simulator/src/math/catmull_rom_spline.cpp [hunk 1]
```diff
@@ -127,19 +119,14 @@ const std::vector<geometry_msgs::msg::Point> CatmullRomSpline::getTrajectory(
...
-
 CatmullRomSpline::CatmullRomSpline(const std::vector<geometry_msgs::msg::Point> & control_points)
 : control_points(control_points)
 {
   size_t n = control_points.size() - 1;
   if (control_points.size() <= 2) {
-    THROW_SEMANTIC_ERROR("numbers of control points are not enough.");
+    THROW_SEMANTIC_ERROR(
+      control_points.size(),
+      " control points are only exists. At minimin, 2 control points are required");
```
Comment (yamacir-kit) [inline simulation/traffic_simulator/src/math/catmull_rom_spline.cpp:129]: minmin

Comment (yamacir-kit) [inline simulation/traffic_simulator/src/math/catmull_rom_spline.cpp:129]: minmin!

**Review by yamacir-kit — CHANGES_REQUESTED** [review]

**Comment by kmiya** [comment]
Yet another missing typo
image

**Review by yamacir-kit — APPROVED** [review]

File: simulation/traffic_simulator/src/math/catmull_rom_spline.cpp (hunk 1 above)
Comment (hakuturu583) [reply simulation/traffic_simulator/src/math/catmull_rom_spline.cpp]: I fixed in 6067310.
Thnaks!

**Pull request merged**